package.json

```diff
@@ -77,7 +77,7 @@
   "private": true,
   "scripts": {
     "build": "scripts/run-script.js --parallel build",
-    "ci": "yarn && yarn test-integration",
```

Comment:
I think this was intended to not build everything to keep the tests as fast as possible.

I think packages required for tests are usually built with a `postinstall` :thinking:

Reply:
I didn't keep the output, but at least  @xen-orchestra/fs wasn't built correctly